src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -1100,7 +1130,11 @@ module Types = struct
                    | Token_owned _ ->
                        false
                    | Not_owned {account_disabled} ->
-                       account_disabled ) ] ))
+                       account_disabled )
+             ; field "index" ~typ:int
+                 ~doc:"The index of this account in the ledger"
```

Comment:
Can you add to the doc the cases when it is None? (in this case when the best_tip_ledger isn't Active, but also try an explain when this happens in practice from an operator's perspective)